### PR TITLE
First tablet version

### PR DESCRIPTION
**Pull request by VinizeraHub**

### CODE REVIEW

File: src/components/common/cards/CaseStudyCard/CaseStudyCard.module.scss [hunk 1]
```diff
@@ -106,7 +107,36 @@
...
+        right: -($container-size - $card-decorator-width) / 2;
+      }
+
+      margin: 0;
+
     }
+    // @include tablet {
+    //   width: 70px;
+    //   height: 70px;
+    // }
```
Comment (BelfDev) [inline src/components/common/cards/CaseStudyCard/CaseStudyCard.module.scss:139]: Can we remove this when shipping the final PR?

File: src/components/pages/HomePage/HomePage.module.scss [hunk 2]
```diff
@@ -117,11 +127,18 @@
         padding-right: 40px;
         padding-left: 40px;
 
-        max-font-size: $typography-desktop-h5;
+        // max-font-size: $typography-desktop-h5;
```
Comment (BelfDev) [inline src/components/pages/HomePage/HomePage.module.scss:130]: Can we remove this when shipping the final PR?

File: src/hooks/useMediaBreakpoints.js [hunk 3]
```diff
@@ -2,13 +2,13 @@ import { useMediaQuery } from 'react-responsive';
 
 export default () => {
   const isMobile = useMediaQuery({
-    query: '(max-width: 768px)',
+    query: '(max-width: 767px)',
   });
   const isTablet = useMediaQuery({
-    query: '(min-width: 769px) and (max-width: 1023px)',
+    query: '(min-width: 768px) and (max-width: 1023px)',
```
Comment (BelfDev) [inline src/hooks/useMediaBreakpoints.js:8]: I strongly suggest we don't go down this path. Overlapping Tablet and Desktop can lead to several mistakes in the future:

- New engineers or our future selves might forget that changes to desktop also influence tablets!
- When looking at a CSS chunk of code, we'll never know what is particular to desktop or tablet
- We'll never be able to make quick changes without double-checking side-effects in tablet platforms
- Tablet and Smartphones are considered Mobile platforms. Overlapping tablet with desktop breaks this intuition.

Suggestion: upgrade our [mixin](https://sass-lang.com/documentation/at-rules/mixin) and [inheritance](https://sass-lang.com/guide) usage to reuse code for desktop and tablets while keeping it clear we are doing it!

File: src/components/common/cards/CaseStudyCard/CaseStudyCard.module.scss (hunk 1 above)
Comment (VinizeraHub) [reply src/components/common/cards/CaseStudyCard/CaseStudyCard.module.scss:139]: Yes :)

File: src/components/pages/HomePage/HomePage.module.scss (hunk 2 above)
Comment (VinizeraHub) [reply src/components/pages/HomePage/HomePage.module.scss:130]: Yes :)

File: src/hooks/useMediaBreakpoints.js (hunk 3 above)
Comment (BelfDev) [reply src/hooks/useMediaBreakpoints.js:8]: #44 solves this issue! No more stones on our way :)
@yagotome  @VinizeraHub

File: src/components/common/lists/HorizontalList/HorizontalList.js [hunk 4]
```diff
@@ -1,21 +1,34 @@
...
-  return (
-    <div className={classNames(styles.horizontalList, className)}>
-      {isDesktop ? (
-        children
-      ) : (
-        <div className={styles.container}>{children}</div>
-      )}
-    </div>
-  );
+const HorizontalList = ({ className, children, variant }) => {
```
Comment (BelfDev) [inline src/components/common/lists/HorizontalList/HorizontalList.js:13]: 👍

**Pull request merged**